web/satellite/src/components/project/ProjectDashboard.vue

```diff
@@ -3,7 +3,14 @@
 
 <template>
     <div class="dashboard-area">
-        <h1 class="dashboard-area__title">{{projectName}} Dashboard</h1>
+        <div class="dashboard-area__header-wrapper">
+            <h1 class="dashboard-area__title">{{projectName}} Dashboard</h1>
+            <VInfo
+                class="dashboard-area__tooltip__wrapper"
+                bold-text="Expect an hour delay between network activity and the latest dashboard stats.">
```

Comment:
I dont think an hour is sufficient here. Lets be more general here.

Reply:
@stefanbenten @mbouzi  the main purpose of this message is to convey to the user that there is a lag; let's update the line to: 
"Expect about an hour delay between network activity and the latest dashboard stats."

Reply:
An hour is not correct. It can be several hours depending on size and load @iglesiasbrandon

Reply:
@mbouzi can we update it to: 

"Expect a delay of a few hours between network activity and the latest dashboard stats."